src/ui/webeiddialog.cpp

```diff
@@ -81,29 +82,59 @@ WebEidDialog::WebEidDialog(QWidget* parent) : WebEidUI(parent), ui(new Private)
     }
     setWindowFlag(Qt::CustomizeWindowHint);
     setWindowFlag(Qt::WindowTitleHint);
-    setWindowTitle(qApp->applicationDisplayName());
-    setTrText(ui->aboutVersion, [] { return tr("Version: %1").arg(qApp->applicationVersion()); });
-    QMenu* langMenu = new QMenu(ui->langButton);
-    langMenu->addAction(QStringLiteral("EN"));
-    langMenu->addAction(QStringLiteral("ET"));
-    langMenu->addAction(QStringLiteral("FI"));
-    langMenu->addAction(QStringLiteral("HR"));
-    langMenu->addAction(QStringLiteral("RU"));
-    connect(langMenu, &QMenu::triggered, qApp, [this](QAction* action) {
-        QSettings().setValue(QStringLiteral("lang"), action->text().toLower());
-        ui->langButton->setText(action->text());
-        qApp->loadTranslations();
-    });
-#ifdef Q_OS_LINUX
-    setStyleSheet(styleSheet() + QStringLiteral("#langButton {padding-right: 15px;}"));
-    connect(ui->langButton, &QToolButton::clicked, this, [this, langMenu] {
-        langMenu->exec(mapToGlobal(ui->langButton->geometry().bottomLeft()));
+    setWindowTitle(QApplication::applicationDisplayName());
+    setTrText(ui->aboutVersion,
+              [] { return tr("Version: %1").arg(QApplication::applicationVersion()); });
+
+    ui->langButton = new QToolButton(this);
+    ui->langButton->setObjectName("langButton");
+    static const std::vector<std::pair<QString, QString>> LANG_LIST {
+        {QStringLiteral("et"), QStringLiteral("Eesti")},
+        {QStringLiteral("en"), QStringLiteral("English")},
+        {QStringLiteral("ru"), QStringLiteral("Русский")},
+        {QStringLiteral("fi"), QStringLiteral("Finnish")},
+        {QStringLiteral("hr"), QStringLiteral("Hrvatska")}};
+    ui->langButton->setText(tr("EN", "ACTIVE_LANGUAGE"));
+    ui->langButton->setAccessibleName(QStringLiteral("English"));
```

Comment:
Is this special case needed? `setAccessibleName()` is called inside `if (auto i = std::find_if(...` below anyway?
```suggestion
```
I'll remove it for now, let's add it back if needed.